examples/js/example12.js

```diff
@@ -16,7 +16,6 @@ L.TimeDimension.Layer.SODAHeatMap = L.TimeDimension.Layer.extend({
             lngField: 'lng',
             valueField: 'count'
         };
-        heatmapCfg = $.extend({}, heatmapCfg, options.heatmatOptions || {});
```

Comment:
I will add a new function inside L.TimeDimension.Util to merge the default configuration with the options (this function: https://gomakethings.com/vanilla-javascript-version-of-jquery-extend/)

Btw, I have this example working after replacing `$.getJSON` by `XMLHttpRequest`

Reply:
makes sense, didn't bother replacing it here, as it was just an example and overriding configuration wasn't used as far as I could see.

Reply:
[Object.assign](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/assign) ? It is supported on all major browser and can be polyfill if needed

Reply:
btw. it is not yet necessary to implement either a polyfill and/or use Object assign. This example is the only place where I removed an `$.extend` bluntly, I replaced it with a simple `for`-loop here (see also https://stackoverflow.com/questions/171251/how-can-i-merge-properties-of-two-javascript-objects-dynamically#171256 for some discussion): https://github.com/socib/Leaflet.TimeDimension/pull/119/commits/ff96af7393b1f5c7c01c877f12f3b5be98e84a85#diff-1a71af7bca9eb8c6869b9fbf22fc0a1bL441 ...